src/Graphing.hs

```diff
@@ -68,3 +90,16 @@ unfold seed getDeps toDependency = foldr addNode empty seed
     where
     children = getDeps dep
     res = toDependency dep
+
+-- | Build a graphing from a list, where all list elements are treated as direct
+-- dependencies
+fromList :: Ord ty => [ty] -> Graphing ty
```

Comment:
This is a new convenience method for building Graphings. Rather than using `unfold` with `const []` as an argument:
```hs
buildGraph nodes = Graphing.unfold nodes (const []) toDependency
```
we instead can do:
```hs
buildGraph nodes = Graphing.fromList (map toDependency nodes)
```

or sometimes point-free:
```hs
buildGraph = Graphing.fromList . map toDependency
```

---

I updated the previous callsites for `unfold` to use `fromList` instead, wherever possible